recipe/meta.yaml

```diff
@@ -1,8 +1,6 @@
-{% set version = "0.11.7" %}
```

Comment:
Removing this is causing a fail when the recipe tries to render (see comment below )